ffka-gluon-web-usb-wan-hotplug/README.md

```diff
@@ -0,0 +1,9 @@
+# ffka-gluon-web-usb-wan-hotplug
+
+This package adds a new option to the config mode, where you can enable to use
+a USB connected Ethernet adapter as a WAN.
```

Comment:
>This package adds an option to the Advanced Settings allowing a user to enable the automatic utilization of USB Ethernet adapters as WAN interfaces. A USB Ethernet adapter can be a Phone with enabled USB tethering as well as Physical USB to RJ-45 adapters.

I wouldn't go for more. This package is ONLY about the UI component whilst the functionality itself is not provided by this package.